stdlib/public/SDK/CoreImage/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+cmake_minimum_required(VERSION 3.4.3)
+include("../../../../cmake/modules/StandaloneOverlay.cmake")
```

Comment:
It's in the ``StandaloneOverlay.cmake`` file, line 42.

``build-script`` and ``build-script-impl`` don't really need to exist. The Swift compiler should be a standalone cmake module that any other project could include if they wanted, like other projects that use cmake.

The problem with the main ``CMakeLists.txt`` file is it sets up so much logic. I would prefer all the logic be in ``cmake/modules/`` or a cmake cache file and have the ``CMakeLists.txt`` basically just include directories recursively based on calculated flags from the modules.